core/field_label.ts

```diff
@@ -131,4 +130,19 @@ export class FieldLabel extends Field {
 
 fieldRegistry.register('field_label', FieldLabel);
 
-(FieldLabel.prototype as AnyDuringMigration).DEFAULT_VALUE = '';
\ No newline at end of file
+(FieldLabel.prototype as AnyDuringMigration).DEFAULT_VALUE = '';
+
+/**
+ * Config options for the label field.
+ */
+export interface Config extends BaseFieldConfig {
+  class
```

Comment:
Formatting seems off - not sure if the formatter got confused by treating class as a keyword?